uaclient/apt.py

```diff
@@ -455,14 +455,18 @@ def clean_apt_files(*, _entitlements=None):
             continue
         repo_file = ent_cls.repo_list_file_tmpl.format(name=ent_cls.name)
         pref_file = ent_cls.repo_pref_file_tmpl.format(name=ent_cls.name)
-
         if os.path.exists(repo_file):
-            logging.info("Removing apt source file: %s", repo_file)
-            os.unlink(repo_file)
-
+            print(
```

Comment:
Even though we are only calling this on function on `prerm`, I think this should be an `event.info` call just for consistency sake

Reply:
@lucasmoura good point, updated!